pkg/controller/mongodb/replica_set_controller.go

```diff
@@ -262,6 +283,7 @@ func (r *ReplicaSetReconciler) Reconcile(request reconcile.Request) (reconcile.R
 	if err := r.resetStatefulSetUpdateStrategy(mdb); err != nil {
 		return status.Update(r.client.Status(), &mdb,
 			statusOptions().
+				withAutomationConfigMembers(mdb.AutomationConfigMembersThisReconciliation()).
```

Comment:
Currently in the status we have members for "statefulset" (kubernetes) and "replicaset" (mongodb), but this API for the statusOptions builder accepts a "withAutomationConfigMembers" which I'm not sure which one it corresponds.